ros2bag/ros2bag/verb/play.py

```diff
@@ -50,8 +50,7 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='rate at which to play back messages. Valid range > 0.0.')
         parser.add_argument(
             '--topics', type=str, default=[], nargs='+',
-            help='topics to replay, separated by space. If none specified, all topics will be '
-                 'replayed.')
+            help='topics to replay, separated by space. At least one topic needs to be specified.')
```

Comment:
```suggestion
            help='Space-delimited list of topics to play.')
```

Bikeshed: I think the language should be like this. `ArgumentParser` will already validate input and provide informative `--help` given the `nargs`.